### DISCUSSION TITLE
中考分数?

### DISCUSSION THREAD

**Discussion by boomzero** [XMOJ-Script-dev/XMOJ-Script#551] [General]
@langningchen @PythonSmall-Q ?

**Comment by boomzero** [comment]
![](https://assets.xmoj-bbs.tech/GetImage?ImageID=mcrulqxilllumaqcgnrkbaaxpgjmbugf)

**Comment by boomzero** [comment]
非常没有实力

**Comment by PythonSmall-Q** [comment]
草我693.5你叫啥

**Reply by boomzero** [reply]
@PythonSmall-Q cln 702

**Comment by PythonSmall-Q** [comment]
英语明天去复核

**Reply by boomzero** [reply]
...

**Comment by langningchen** [comment]
@PythonSmall-Q 希望你能通过

**Comment by boomzero** [comment]
And?

**Comment by PythonSmall-Q** [comment]
分数没动，释怀了
















在 2024-07-13 18:29:48，"Zhu Chenrui" ***@***.***> 写道：

And?

**Comment by langningchen** [comment]
What a pity 😞

**Comment by boomzero** [comment]
Luckily it doesn’t matter
